package.json

```diff
@@ -100,7 +100,7 @@
   "dependencies": {
     "@across-protocol/across-token": "^1.0.0",
     "@across-protocol/constants": "^3.1.32",
-    "@across-protocol/contracts": "3.0.26-beta.1",
+    "@across-protocol/contracts": "4.0.0-beta.3",
```

Comment:
Bump to 4.0.0 now?

Reply:
https://github.com/across-protocol/sdk/pull/829/commits/11cf3beb9af2330fa1e84f46dee7cc8df62c4531